### DISCUSSION TITLE
Some user interface enhancements?

### DISCUSSION THREAD

**Discussion by jjojala** [OpenOrienteering/mapper#2212] [Ideas]
I use Mapper a lot and like it too. But surely there're always room for improvements, right?

Here's few minor(?) user interface issues that annoy me, how's others?

I know, some of these are "inherited" from OCAD, but it's not necessary to imitate every feature from it, although it at least superficially helps adopting Mapper by OCAD users. Some of these may also be related to my personal style of using Mapper.

### "Overprint simulation" shortcut key issue on desktop use

On desktop, using "Hatch area" (F2), "Baseline view" (F3) are frequent, but "Overprinting simulation" (F4) is rare. As map editing obviously concentrates on using mouse (and 10-finger -style keyboard use does not apply), risk of mispunching F4 is likely (especially as function key layout slightly differs depending on keyboard). 

As whether the "Overprinting simulation" is on or off is not directly easy to see, and as it slows down screen rendering, mispunching F4 is annoying. As I claimed above, real use is rare.

My proposal is that "Overprinting simulation" would be usable only through menus or moved to some other shortcut. Of course shortcuts could be configurable as well.

### Disable all templates function

While obviously providing necessarily information for mapping, templates do occasionally disrupt to see the appearance of the produced map. Would be very beneficial to have dedicated button ("shortcut") to disable all templates on touch screen use as well! Disabling/enabling templates through "template control" button is far too impractical.

The need for this even increases if/when one may want use several templates at the same time. Perhaps "enable/disable all templates" toggle button should be close to "Hatch area" and "Baseline" toggles, as to me it's kind of on par with them... 

On the desktop use, it is also (/could be used more) frequently used, but as it is on the right hand side of the keyboard, I'd use it my right hand... which is occupied by the mouse... This is of course "problem" only if you want to use mouse with your right hand.

"Disable all templates" -toggle could be moved e.g. to F4 so that one can use it on the left hand... (and as it is kind of on par with F3 and F4...) or it can be configure...

Example of "Terrain Ruggedness Index" -image, usefull on the field, but unfortunately interferes to see the resulting map (therefore toggling templates would be usefull)
![tri-sample](https://github.com/OpenOrienteering/mapper/assets/6313950/8ae450fb-5b2d-4e2b-8812-1c52d7ec9834)

### Del/Edit buttons on touch mode

On touch screen these are close each other. Minor and easy to fix (with undo), but sometimes annoying.

### "Draw circles and ellipses" tool on mouse menu

Quite many natural map features are handy to make by first drawing a ellipse and then re-shaping it to a correct form. Most natural features rarely have sharp corners. Therefore to me it should be on mouse (right click) menu (at least preferred when comparing to "Draw squares" tool - which is there).

**Comment by dl3sdo** [comment]
I also see no need for an "Overprinting simulation" shortcut.
